sql/item_mods.sql

```diff
@@ -52119,6 +52119,18 @@ INSERT INTO `item_mods` VALUES (23795,1,1); -- DEF: 1
 -- Ziamet Nails
 INSERT INTO `item_mods` VALUES (23796,1,1); -- DEF: 1
 
+-- Poroggo Cass. +1
+INSERT INTO `item_mods` VALUES (23804,15,20);   -- FIRE_RES: 20
+INSERT INTO `item_mods` VALUES (23804,16,20);   -- ICE_RES: 20
+INSERT INTO `item_mods` VALUES (23804,17,20);   -- WIND_RES: 20
+INSERT INTO `item_mods` VALUES (23804,18,20);   -- EARTH_RES: 20
+INSERT INTO `item_mods` VALUES (23804,19,20);   -- THUNDER_RES: 20
+INSERT INTO `item_mods` VALUES (23804,20,20);   -- WATER_RES: 20
+INSERT INTO `item_mods` VALUES (23804,21,20);   -- LIGHT_RES: 20
+INSERT INTO `item_mods` VALUES (23804,22,20);   -- DARK_RES: 20
```

Comment:
All of this should be AILMENT resistances, not element

Reply:
Thank you for noticing I will get right on this!

Reply:
ok this is all set